packages/core/src/components/meter/meter.scss

```
@@ -0,0 +1,86 @@
+:host {
+  align-items: flex-start;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  width: 100%;
+  word-wrap: break-word;
+}
+
+.container-text {
+  align-items: flex-end;
+  display: flex;
+  font: var(--text-body-small);
+  font-weight: 300;
+  justify-content: space-between;
+  letter-spacing: var(--text-body-small-letter);
+  margin-bottom: var(--spacing-xsmall);
+  width: 100%;
+
+  &.hasCenterTitle {
+    justify-content: center;
+  }
+
+  &.hasMarginTop {
```

block: out of our patterns, also is not a good idea to use names of properties to define props, `gap-top` is better now


```suggestion
  &.has-gap-top {
```

---

Nice observation!

Made: https://github.com/juntossomosmais/atomium/pull/695/commits/77a58917a2735eda1a41abaa8309c320a481b590